node/mock-task.ts

```diff
@@ -64,6 +62,32 @@ module.exports.getBoolInput = task.getBoolInput;
 module.exports.getDelimitedInput = task.getDelimitedInput;
 module.exports.filePathSupplied = task.filePathSupplied;
 
+export function getVariable(name: string): string | undefined  {
+    const variables = mock.getVariableMap();
```

Comment:
Lets check if `name` is null/undefined and [throw if so](https://github.com/Microsoft/azure-pipelines-task-lib/blob/11c9439d4af17e6475d9fe058e6b2e03914d17e6/node/internal.ts#L256).

Reply:
Resolved. `getVariable` is now calling `im._getVariableKey` to throw or normalize.